sktime/benchmarking/benchmarks.py

```diff
@@ -10,6 +10,39 @@
 from sktime.utils.validation._dependencies import _check_soft_dependencies
 
 
+def coer_estimator_and_id(estimators, estimator_id=None):
```

Comment:
Real nitpicky non-blocker suggestion: can you rename it to `coerce` from `coer` (in all relevant places)? Personally as a user/developer I'll appreciate seeing 2 more characters to readily understand what the function is supposed to do.

Reply:
ofcourse! you are right!